release_notes/topics/8_0_0.adoc

```diff
@@ -45,6 +45,18 @@ In this release, we added initial support for W3C Web Authentication (WebAuthn).
 however we are working on further improvements in this area. Thanks to https://github.com/tnorimat[tnorimat] for the contribution. Also thanks to
 https://github.com/ynojima[ynojima] for the help and feedback.
 
+== Support for password-less authentication, multi-factor authentication and multiple credentials per user
+
+With the arrival of W3C Web Authentication support, we've refined the authentication flow system to be able to allow a user to select which authentication method he
+wishes to use during his login (for example, to choose between his OTP credential and his WebAuthn credential). The new mechanisms also allow an administrator to
```

Comment:
change "his OTP credential and his WebAuthn" to "the OTP credential and the WebAuthn"

Reply:
changing to: "(for example, the choice between an OTP credential and a WebAuthn credential)"